example/google.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { MapContainer, TileLayer, LayersControl } from 'react-leaflet'
-import { GoogleLayer } from '../src'
+//import { GoogleLayer } from '../src'
```

Comment:
please remove that

Reply:
it's there so you can see the error when we run this by importing GoogleLayer from dist when we import it from src there is no error